src/mainwindow/mainwindow.cpp

```diff
@@ -561,14 +553,15 @@ void MainWindow::onConnectTriggered()
     {
         return;
     }
-    mCurrentUserName = userInput;
     mpChatWindowDock->setUserName(userInput);
-    mplocalConnector->configureOnLogin(mCurrentUserName);
+    mplocalConnector->configureOnLogin(userInput);
 }
 
 void MainWindow::onSettingsTriggered()
 {
     MenuOptions * menuOptions = new MenuOptions(this);
```

Comment:
MenuOptions *menuOptions = new MenuOptions(this);

Reply:
It is Roman's code